packages/unlock-js/package.json

```diff
@@ -28,7 +28,7 @@
   "author": "Unlock Inc",
   "license": "MIT",
   "peerDependencies": {
-    "ethers": "5.7.2"
+    "ethers": "^5.7.2"
```

Comment:
Also we could do ^5.x.x so so we support any 5 version of ethers. 

Depends on what features we use internally of ethers.

Reply:
fixed!